session/session.go

```diff
@@ -252,9 +253,9 @@ func (s *session) OpenCh(pctx context.Context, openingBalInfo perun.BalInfo, app
 	}
 
 	sanitizeBalInfo(openingBalInfo)
-	parts, err := retrieveParts(openingBalInfo.Parts, s.contacts)
+	parts, err := retrieveParts(openingBalInfo.Parts, s.idProvider)
 	if err != nil {
-		s.Error(err, "retrieving channel participants using session contacts")
+		s.Error(err, "retrieving channel participants using session idProvider")
 		return perun.ChInfo{}, perun.GetAPIError(err)
 	}
 	registerParts(parts, s.chClient)
```

Comment:
```suggestion
	registerPartIDs(partIDs, s.chClient)
```

As we renamed Peer to PeerID, renaming these variables accordingly.
Please update the variable names in other places of usage as well.